velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -224,60 +297,91 @@ HiveDataSink::HiveDataSink(
       insertTableHandle_(std::move(insertTableHandle)),
       connectorQueryCtx_(connectorQueryCtx),
       commitStrategy_(commitStrategy),
+      maxOpenWriters_(
+          HiveConfig::maxPartitionsPerWriters(connectorQueryCtx_->config())),
```

Comment:
Do we need to update documentation for this configuration property to clarify that it applies to both partitions and buckets?

Reply:
Updated. Thanks!